.github/workflows/build.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           SHALLOW: 1
       - name: Check out jspecify/samples-google-prototype
         run: git -C ../jspecify checkout samples-google-prototype
+        if: always()
```

Comment:
I think the `if` should come near the top, just after the `name`, since it's a guard condition. (It obviously works correctly no matter where, but I think it's a bit more readable at the top.)

Reply:
Done.